src/frontend/apps/impress/src/features/docs/doc-editor/styles.tsx

```diff
@@ -88,7 +88,7 @@ export const cssEditor = (readonly: boolean) => css`
       font-size: 1.25rem;
     }
     a {
-      color: var(--c--theme--colors--greyscale-500);
+      color: var(--c--theme--colors--greyscale-700);
```

Comment:
If you add 700 you will not have any color distinction between your text and your link on the other hand:
With 500 (as before): 
<img width="343" height="75" alt="image" src="https://github.com/user-attachments/assets/9a8c1ecd-2c3b-413f-89d3-00452f914c52" />
With 700 (new change):
<img width="277" height="47" alt="image" src="https://github.com/user-attachments/assets/e2543e19-db42-4d57-bcef-660e014eebf3" />

I am fine with 700 if you take that into account.